Userland/Libraries/LibWeb/Loader/ResourceLoader.cpp

```diff
@@ -192,6 +192,14 @@ static void log_failure(LoadRequest const& request, ErrorType const& error)
     dbgln("ResourceLoader: Failed load of: \"{}\", \033[31;1mError: {}\033[0m, Duration: {}ms", url_for_logging, error, load_time_ms);
 }
 
+static void log_filtered_request(LoadRequest const& request)
+{
+    auto url_for_logging = sanitized_url_for_logging(request.url());
+
+    emit_signpost(ByteString::formatted("Filtered request: {}", url_for_logging), request.id());
```

Comment:
`emit_signpost` seems to be a holdover from SerenityOS that no longer does anything, so I don't believe it's needed here.

Reply:
Removed emit_signpost.